### DISCUSSION TITLE
What we learned

### DISCUSSION THREAD

**Discussion by xavier506** [eoscostarica/gGoods#200] [General]
## What we learned

- NFTs can be a new platform for crowdfunding
- NGOs and communities are looking for new ways to engage with their supporters and raise funds
- People are looking for new ways to support causes they care about through an experience that doesn't involve physical travel or just a monetary donation
- NFTs can be more impactful if they are not just collectible art. By using them as in-app assets, they become a badge of honor, a representation of the user and a fun digital asset.

**Comment by kriskoin** [comment]
i learned how to use dgoods https://dgoods.org/ as a NFT framework
